<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="when_dict_update" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>659</width>
      <height>78</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="setHidden" stdset="0">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="renewDict" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>监测到词典改动，是否更新？                  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renewDict_ok_btn">
           <property name="text">
            <string>是</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renewDict_cancel_btn">
           <property name="text">
            <string>否</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="updateDict" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>480</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="update_info">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>831</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent; 
font: 75 15pt &quot;Gill Sans&quot;;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gill Sans'; font-size:15pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Helvetica Neue'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_dict">
    <property name="title">
     <string>词典维护</string>
    </property>
    <addaction name="actionaddNewWord"/>
    <addaction name="separator"/>
    <addaction name="actionremoveWord"/>
    <addaction name="separator"/>
    <addaction name="actionchangeWord"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_list">
    <property name="title">
     <string>索引维护</string>
    </property>
    <addaction name="actionreadDict"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_resolve">
    <property name="title">
     <string>处理文件</string>
    </property>
    <addaction name="actionreadFromFile"/>
    <addaction name="separator"/>
    <addaction name="actionreadFromInput"/>
   </widget>
   <addaction name="menu_dict"/>
   <addaction name="menu_list"/>
   <addaction name="menu_resolve"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionaddNewWord">
   <property name="text">
    <string>新增词条</string>
   </property>
   <property name="toolTip">
    <string>新增</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionreadFromFile">
   <property name="text">
    <string>导入文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionremoveWord">
   <property name="text">
    <string>删除词条</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionchangeWord">
   <property name="text">
    <string>修改词条</string>
   </property>
   <property name="toolTip">
    <string>修改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionreadFromInput">
   <property name="text">
    <string>手动输入</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionreadDict">
   <property name="text">
    <string>读取词典</string>
   </property>
   <property name="toolTip">
    <string>读取</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
